src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visits/VisitsMigrationService.kt

```diff
@@ -74,37 +72,54 @@ class VisitsMigrationService(
       }
       ?: run {
         val nomisVisit = nomisApiService.getVisit(context.body.visitId)
-        determineRoomMapping(nomisVisit)
```

Comment:
refactored to not use null as the control flow but response object instead to be explicit - it was getting very confusing